cmd/worklog.go

```diff
@@ -175,6 +176,19 @@ func worklogResolve(ctx *cli.Context) error {
 
 	w := tabwriter.NewWriter(os.Stdout, 2, 0, 2, ' ', 0)
 	for _, item := range toResolve {
+		if item.Type() == apitypes.InviteApproveWorklogType {
+			w.Flush()
+
+			err = AskPerform("Approve invite for " + item.Subject)
```

Comment:
We should think about how we want to display, perhaps - "Fullname (username; email)" ?

We can also open an issue and make this standard throughout the invite bits after this is merged. Right now though we display it in a ton of different ways, so it's not clear where people come from or which invite has led to which user joining.